### DISCUSSION TITLE
Quản lý thông tin sách - Thêm , Xoá , Sửa

### DISCUSSION THREAD

**Discussion by duyan0** [duyan0/WebsiteBook#38] [Announcements]
khi thêm sách thì không nhận dữ liệu từ tác giả

**Comment by Awan2004** [comment]
Lỗi NamXB

**Comment by duyan0** [comment]
**Không thể xoá sách**

**Comment by duyan0** [comment]
Vẫn chưa fix được

**Comment by TanDat0612** [comment]
Thì thôi

Vào Th 7, 9 thg 11, 2024 lúc 10:21 yjendev ***@***.***> đã
viết:

**Reply by duyan0** [reply]
chịu thôi
